src/script/zcash_script.cpp

```diff
@@ -218,6 +218,38 @@ unsigned int zcash_script_legacy_sigop_count(
     }
 }
 
+void zcash_script_transparent_output_address(
+    const unsigned char *txTo,
+    unsigned int txToLen,
+    unsigned int nOut,
+    unsigned char *address,
```

Comment:
Naming convention for output pointers

```suggestion
    unsigned char *addressOut,
```

Reply:
No longer needed, address is now returned